### PR TITLE
EUI-4210-from-master

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4210

### Change description ###
Automated test failure

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by udaydenduluri33 — DISMISSED** [review]
@Mo-Lala84 Approved

**Review by leepavlouhmcts — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "4.3.0",
+    "@hmcts/ccd-case-ui-toolkit": "4.3.4-download-secure-docstore",
```
Comment (udaydenduluri33) [inline package.json:72]: @RiteshHMCTS  need a full version with semantic versioning

File: src/cases/resolvers/activity.resolver.ts [hunk 2]
```diff
@@ -9,7 +9,11 @@ export class ActivityResolver implements Resolve<boolean> {
   constructor(private readonly activityService: ActivityService) {}
 
   public resolve(): Observable<boolean> {
-    this.activityService.verifyUserIsAuthorized();
+    try {
+      this.activityService.verifyUserIsAuthorized();
+    } catch (err) {
+      console.log(err);
```
Comment (udaydenduluri33) [inline src/cases/resolvers/activity.resolver.ts:15]: Why are we logging and eating away this exception?

Comment (udaydenduluri33) [reply src/cases/resolvers/activity.resolver.ts:15]: `return this.activityService.verifyUserIsAuthorized();`

**Review by udaydenduluri33 — APPROVED** [review]
Agreed with @sreekanthpuligadda  and @RiteshHMCTS  that eating away exception is not ideal. But, the change will be done later

**Pull request merged**